src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1610,10 +1634,10 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
                 }
 
                 // May have media/type; charset=utf-8, allow partial match.
-                if (!Request.ContentType.StartsWith("application/x-www-form-urlencoded", StringComparison.OrdinalIgnoreCase)) {
+                if (Request.ContentType.IndexOf("application/x-www-form-urlencoded", StringComparison.OrdinalIgnoreCase) != -1) {
```

Comment:
Requiring that the content-type start with `application/x-www-form-urlencoded` instead of containing it implies that a consumer of the endpoint will never put a media/type before `application/x-www-form-urlencoded`; for true partial matches in the `Request.ContentType` I thought it might be better to allow that media/type to be anywhere in the string, not just at the beginning.